.release-plan.json

```diff
@@ -1,16 +1,7 @@
 {
   "solution": {
     "@embroider/addon-dev": {
-      "impact": "patch",
-      "oldVersion": "3.1.0",
-      "newVersion": "3.1.1",
-      "constraints": [
-        {
-          "impact": "patch",
-          "reason": "Appears in changelog section :bug: Bug Fix"
-        }
-      ],
-      "pkgJSONPath": "/Users/edward/hacking/embroider/packages/addon-dev/package.json"
```

Comment:
why include the `pkgJSONPath` here? (since it's machine specific?)

Reply:
It's actually needed, but probably that means we have a bug and right now you need to run `prepare` and `publish` on the same machine.